assets/settings/default.json

```diff
@@ -1,6 +1,18 @@
 {
   // The name of the Zed theme to use for the UI
   "theme": "One Dark",
+  // The theme can also be set to follow system preferences.
```

Comment:
Could we move this comment up above the `theme` field?

Reply:
Absolutely! Is it okay if this is in a seperate commit?

Reply:
> Absolutely! Is it okay if this is in a seperate commit?

Yep, that's fine.

We squash commits when merging to `main`, so it all ends up in a single commit anyways 🙂

Reply:
Ok, great! I've submitted a new commit for this, do I need to do anything more in this PR to indicate that the change is made? I'm not too familiar with submitting PR:s here :)